### ISSUE TITLE
[BUG] Default do a restart if any other instance is running

### ISSUE THREAD

**Issue by Hofknecht** [Hofknecht/SystemTrayMenu#320]
in #276 we introduced an argument -h or -hotkey that when there is already another instance running then send hotkey instead of killing the other instance

in #305 we made this option default true and introduced an argument -r that when there is already another instance running (for the old behavior)

but if application is hanging due to a bug it is important that we default restart (kill other instances) instead of sending hotkey to other instance
![2022-02-07 16_24_53-Window](https://user-images.githubusercontent.com/52528841/152817669-f65ca535-8e1b-4d01-973f-b5462bdbf6be.png)

and also let's do as an checkbox in options instead as argument
![2022-02-07 16_30_39-Window](https://user-images.githubusercontent.com/52528841/152819042-cb0aede1-dc26-4ee0-bb44-aa9002d8548c.png)

**Comment by chip33** [comment]
I saw that you see reports from users and try to fix them, one tiny thing I noticed is that sometimes "Store config at the assembly location" does not place a user.config there and the tick does not stick in the settings box.
I think this is due to windows permissions blocking it when it is located in a protected location like program files (where windows will not allow you to say convert a file to a zipfile).
If you have STM somewhere where you can have a user.config in assembly location and then move it with the user.config inside to a protected location you can sometimes get an error and the mail app opens.

**Comment by Hofknecht** [comment]
Yes, I am not quite sure what exactly the problem with the comment is or was, but this should be ruled out better, we have solved the bugs, but it should also be better for future issues to be the default behavior to restart

A common reason why it doesn't start is of course that the antivirus deletes it, that could of course also be the case.

Just released that point here to prevent further bad comments in this direction.

=> so current issue here close

What you mention would then be something that we should follow up on extra, we can open an issue for that. Can you reproduce this, maybe again document it via video and if it is the case create a issue

**Comment by chip33** [comment]
I have noticed this quite a few times, I have UAC at minimum and run as administrator Windows will not let me for example create a zip of a file or drop a browser bookmark shortcut into any folder in program files.
The easiest way I find to test that strict permissions are in force on that folder is by right clicking a file and trying to make a shortcut of it. If you get this message you know access is restricted.
![Screenshot 2022-02-07 162248](https://user-images.githubusercontent.com/48499740/152828952-d3a6ceb4-f400-459b-8191-1cf6599395b8.png)

Then you should be able to reproduce it by moving the STM assembly location folder with an active user.config inside to the protected folder, launching a new instance of STM from there and when you try to change settings in STM it will crash as it tries to edit the user.config and fails.

![Screenshot 2022-02-07 163018](https://user-images.githubusercontent.com/48499740/152830642-190e8037-b24f-4995-82be-f75f6e4c6190.png)

**Comment by chip33** [comment]
Moved this to new bug report.
